gdcdictionary/schemas/pcr_assay_file.yaml

```diff
@@ -0,0 +1,100 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "pcr_assay_file"
+title: PCR Assay File
+type: object
+namespace: https://www.bloodpac.org
+category: data_file
+program: '*'
+project: '*'
+description: >
+  A data file containing the results of a PCR assay
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+      - name: pcr_assays
+        backref: pcr_assay_files
+        label: data_from
+        target_type: pcr_assay
+        multiplicity: many_to_one
+        required: false
+      - name: analytes
+        backref: pcr_assay_files
+        label: data_from
+        target_type: analyte
+        multiplicity: many_to_one
+        required: false
+
+required:
+  - submitter_id
+  - file_name
+  - file_size
+  - data_format
+  - md5sum
+  - data_category
+  - data_type
+  - experimental_strategy
+
+uniqueKeys:
+  - [ id ]
+  - [ project_id, submitter_id ]
+
+properties:
+  $ref: "_definitions.yaml#/data_file_properties"
+
+  analysis_software:
+    description: >
+      The software used to process PCR assay raw data
+    enum:
+      - "BD Accuri C6"
+      - "Q-Rex"
+      - "QuantaSoft"
+      - "QuantStudio Dx"
+
+  data_category:
+    term:
+      $ref: "_terms.yaml#/data_category"
+    enum:
+      - "Mutant Allele Fractions"
+
+  data_type:
+    term:
+      $ref: "_terms.yaml#/data_type"
+    enum:
+      - "Processed Data"
+      - "Tabulated Results"
+      - "Template"
+      - "Unprocessed Data"
+
+  data_format:
+    term:
+      $ref: "_terms.yaml#/data_format"
+    enum:
+      - "CSV"
+      - "QLB"
+      - "QLT"
+      - "QLP"
+      - "TSV"
+
+  experimental_strategy:
+    description: >
+      The PCR assay strategy used to generate the data file.
```

Comment:
This one has a period. Also there's an experimental strategy in the _terms file although its description says "sequencing strategy". Maybe generalize that and add the term here?